spotify_project/base/urls.py

```diff
@@ -9,4 +9,5 @@
     re_path(r'^callback/$', views.spotify_callback, name='callback'),
     re_path(r'^pull_data/$', views.pull_data, name='pull_data'),
     re_path(r'^all_artists/$', views.all_artists, name='all_artists'),
+    path(r'single_artist/<str:artist_id>/', views.single_artist, name='single_artist'),
```

Comment:
We'll eventually want to move this to `artists.urls` instead. But this is fine for now.